### PR TITLE
Fix #172: Add clean option to $OMEGA_HOME/progs makefiles

### LINKED ISSUE #172: 'Make clean' is not defined for addphsp, statdose (and possibly other user codes)
Recall issue #49.

### PR DESCRIPTION
**Pull request by blakewalters**

### CODE REVIEW

**Comment by ftessier** [comment]
Thanks @blakewalters! Can you please update the commit title to read `Fix #172: ...`. The number sign tells github to add a link to the issue. Usefule when reviewing.

**Comment by ftessier** [comment]
@blakewalters I meant the title of the actual commit. On the `fix-omega_makefiles` branch, issue the `git commit --amend` command to edit the latest commit message and then force push to update the branch here. Thanks!

File: HEN_HOUSE/omega/progs/addphsp/Makefile [hunk 1]
```diff
@@ -45,3 +45,9 @@ addphsp_$(my_machine).f: $(SOURCES)
 	@echo "Mortran compiling addphsp sources ..."
 	@$(MORTRAN_EXE) -d $(MORTRAN_DATA) -f $(SOURCES) -o7 $@ \
 	    -o8 addphsp_$(my_machine).mortlst
+
+clean:
+	$(REMOVE) mortjob.mortran addphsp_$(my_machine).F addphsp_$(my_machine).mortlst
```
Comment (ftessier) [inline HEN_HOUSE/omega/progs/addphsp/Makefile]: @blakewalters this should be `addphsp_$(my_machine).f` (lowercase `f`) I think, according to compilation target above.

File: HEN_HOUSE/omega/progs/dosxyz_show/Makefile [hunk 2]
```diff
@@ -43,13 +43,13 @@ X11_include = -I/usr/X11R6/include
 # On our system, Motif include files are in /usr/X11R6/include/Xm =>
 # we don't need a separate -I for Motif. If this is different on your
 # system, adjust accordingly.
-Motif_include =
+Motif_include = -I/opt/local/include
```
Comment (ftessier) [inline HEN_HOUSE/omega/progs/dosxyz_show/Makefile]: @blakewalters is this additions for the Motif `-I` path intentional?

File: HEN_HOUSE/omega/progs/dosxyz_show/Makefile [hunk 3]
```diff
@@ -43,13 +43,13 @@ X11_include = -I/usr/X11R6/include
...
 
 X11_libdir = -L/usr/X11R6/lib
 # On our system, the Motif library is /usr/X11R6/lib =>
 # we don't need a separate -L for Motif. If this is different on your
 # system, adjust accordingly.
-Motif_libdir =
+Motif_libdir = -L/opt/local/lib
```
Comment (ftessier) [inline HEN_HOUSE/omega/progs/dosxyz_show/Makefile]: @blakewalters is this additions for the Motif `-L` path intentional?

File: HEN_HOUSE/omega/progs/addphsp/Makefile (hunk 1 above)
Comment (blakewalters) [inline HEN_HOUSE/omega/progs/addphsp/Makefile]: You're right!  Changing this now.

File: HEN_HOUSE/omega/progs/dosxyz_show/Makefile (hunk 3 above)
Comment (blakewalters) [inline HEN_HOUSE/omega/progs/dosxyz_show/Makefile]: Must be remnants of an attempted compile here at home.  These should, indeed, be empty.  I'll fix this as well and then update the branch.

Comment (ftessier) [inline HEN_HOUSE/omega/progs/dosxyz_show/Makefile]: Ok, just modify the files and then use `git commit --amend` so it replaces the existing commit, and then `git push --force fix-omega_makefiles`. Or else just push the new commit as usual, and I'll take care of it!

**Comment by blakewalters** [comment]
See the latest commits to this branch.

**Comment by ftessier** [comment]
Ok thanks, I updated the branch to combine everything into one commit and base it on develop.

**Pull request merged**